pandas/core/generic.py

```diff
@@ -2897,6 +2903,10 @@ def to_latex(
         if multirow is None:
             multirow = config.get_option("display.latex.multirow")
 
+        # formatters is a list
+        if formatters_col and type(formatters_col) == list:
```

Comment:
Use `is_list_like` here and add raise a `ValueError` if the use has passed something other than a list.